assets/prefabs/Blowtorch.prefab

```diff
@@ -0,0 +1,28 @@
+{
+  "parent": "engine:iconItem",
+  "Item": {
+    "icon": "Cooking:Blowtorch”,
```

Comment:
Watch out - you've got the wrong kind of double quote in here. That's going to be messing up the file. Several others exist (copy paste?)